docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1c.md

```diff
@@ -0,0 +1,71 @@
+- check if the starting_system_type is in the list of eligible system types: `if starting_system_type in eligible_primary_system_types:`
+	- get the list of all systems and the zones they serve in the RMD by calling get_hvac_zone_list_w_area: `hvac_zone_list_w_area = get_hvac_zone_list_w_area(B_RMD)`
+	- loop through each system in eligible_systems_in_zone: `for eligible_system in eligible_systems_in_zone:`
+		- create a variable to hold the sum total of all peak loads in the system:
+		`system_total_peak_load = 0`
+		- create a variable to hold the sum total of the system area:
+		`system_total_area = hvac_zone_list_w_area[eligible_system.id]["TOTAL_AREA"]`
+		- create a list of the EFLH's for each zone `eflh = []`
+		- get the list of zones associated with this system: `zones_associated_with_system = hvac_zone_list_w_area[eligible_system.id]["ZONE_LIST"]`
+		- loop through the zones in the list: `for zone_id in zones_associated_with_system:`
+			- find the zone using the find_zone function: `zone = find_zone(zone_id)`
+				- check that zone_b is not the zone being investigated:
+				`if zone_b != zone:`
+					- add the zone internal loads to the system_total_peaks:
+					`a = get_zone_peak_internal_load(zone_b)`
+					`system_total_peak += a[0]`
+					- get the zone eflh by using the get_zone_eflh function.  This returns eflh for the year, so we need to convert to weeks by dividing by 52.1428: `zone_eflh = get_zone_eflh(zone)/52.1428`
+					- add the eflh to the eflh list.  We are multiplying the schedule by the zone area so that later we can divide the total by the total area.  Otherwise small zones with very different schedules have too great an impact
+					`eflh.append(zone_eflh * a[1])`
+		- now calculate the average internal load: `avg_internal_load = system_total_peak / system_total_area`
+		- and calculate the averge eflh: `avg_eflh = sum(eflh)/len(eflh)`
+		- get the internal load of the reference zone: `internal_loads = get_zone_peak_internal_load(zone)`
+		- now do the rule checks:
+		- if the zone peak differs by more than 10 btu/hr/sf from the average, then it meets the exception:
+		`if(abs(internal_loads[0]/internal_loads[1] - avg_internal_load) > 10):
+			- it meets the exception: `result = YES`
+		- OR if the eflh differs by more than 40, then the zone meets this exception:
+		`if(abs(internal_loads[2] - eflh) > 40):
+			- it meets the exception: `result = YES`
+		- if the zone is still eligible, check if it is a computer room zone (computer room zones are not eligible for this exception): `if result == YES:`
+			- use the function zones_with_computer_room_dict_x to get a list of zones including computer rooms: `computer_room_zones_dict = zones_with_computer_room_dict_x(B_RMD)`
```

Comment:
It seems no other dependency in this logic, we can move it to the top to avoid the EFLH and peak load processing.

Reply:
agreed.  moved.